From: Amit Pundir <amit.pundir@linaro.org>
Date: Sat, 26 Sep 2020 00:08:29 +0530
Subject: arm64: dts: qcom: sdm845-xiaomi-beryllium-common: Enable ath10k
 host-cap skip quirk

The WiFi firmware used on Beryllium (Xiaomi Poco F1) phone
doesn't support the host-capability QMI request, hence enable
the skip quirk for this device.

Signed-off-by: Amit Pundir <amit.pundir@linaro.org>
Change-Id: If332ff59ea23b7ff3000b7a14269437ec574c084
---
 arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
index 7633d01..b9f9a24 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
@@ -623,6 +623,8 @@ &wifi {
 	vdd-1.3-rfa-supply = <&vreg_l17a_1p3>;
 	vdd-3.3-ch0-supply = <&vreg_l25a_3p3>;
 	vdd-3.3-ch1-supply = <&vreg_l23a_3p3>;
+
+	qcom,snoc-host-cap-skip-quirk;
 };
 
 /* PINCTRL - additions to nodes defined in sdm845.dtsi */
